core/liquivision.c

```diff
@@ -310,6 +310,7 @@ static void parse_dives (int log_version, const unsigned char *buf, unsigned int
 
 		unsigned int event_code, d = 0, e;
 		struct lv_event event;
+		memset(&event, 0, sizeof(event));
```

Comment:
This does not hurt - but the warning is bogus.

The code accessing `event` will only be reached if `event_code == 0xf` in which case `handle_event_ver3()` will have filled out the appropriate fields. Usually gcc is quite smart about these things. I wonder why it is missing it in this case.